src/status_im/ui/screens/events.cljs

```diff
@@ -201,6 +204,36 @@
  (fn [{:keys [db]} [_ enabled?]]
    {:db (assoc db :two-pane-ui-enabled? enabled?)}))
 
+;; This counter is used
```

Comment:
I don't think that belongs here, are you sure you can't use `:component-did-mount` and `component-will-unmount`

btw:

`The event subscriptions are called in reverse order (i.e. last registered subscription first), and if one subscription returns true then subscriptions registered earlier will not be called. Beware: If your app shows an opened Modal`

so there is no point cancelling them `on-blur`


``` clojure
(defn wizarrd []
  (let [back-handler (atom nil)]
    (reagent/create-class
     {:component-did-mount
      (fn []
        (reset! back-handler
                (.addEventListener js-dependencies/back-handler
                                   "hardwareBackPress"
                                   (fn []
                                     (re-frame/dispatch [::multiaccounts.recover/enter-phrase-back-button-pressed])
                                     true))))
      :component-will-unmount
      (fn []
        (.remove @back-handler))
      :reagent-render
      })))
```

Reply:
In this scenario we will end-up with as many back-button listeners as there are components in the navigation stack.

Reply:
Oh, sorry. But then, as you said, subs registered earlier won't get called. Phew :)

Reply:
Moved to intro-wizard's effect handlers. Investigating a solution using built-in react-navigation functions.